crates/solidity/outputs/cargo/crate/src/lib.rs

```diff
@@ -1,3 +1,42 @@
+#![deny(missing_docs)]
+#![deny(rustdoc::broken_intra_doc_links)]
+//! Slang is intended to be a modular Solidity compiler, specifically targeting code analysis and developer tooling.
```

Comment:
This file is effectively the homepage/README on docs.rs .. Should we add a `_PRODUCT_README_` marker and sync its contents here?